conf/evolutions/default/252.sql

```diff
@@ -0,0 +1,129 @@
+# --- !Ups
+
+ALTER TABLE audit_task
+    ADD CONSTRAINT fk_audit_task_amt_assignment
```

Comment:
Instead of naming the foreign keys explicitly, let's just use the names that they give to the constraints by default. This is how we've done it elsewhere, so we should continue with that so that naming conventions aren't changing! I believe that the lines should look more like:
```
ALTER TABLE audit_task
    ADD FOREIGN KEY (amt_assignment_id) REFERENCES amt_assignment(amt_assignment_id);
```

Then it will default to naming the keys `<table-name>_<column-name>_fkey`. So for the first one, it would be `audit_task_amt_assignment_id_fkey`. You'll still need to know what they'll be called for the downs!